README.md

```diff
@@ -249,7 +253,6 @@ Supports following ruby-kafka's producer options.
 - compression_codec - default: nil - The codec the producer uses to compress messages.
 - max_send_limit_bytes - default: nil - Max byte size to send message to avoid MessageSizeTooLarge. For example, if you set 1000000(message.max.bytes in kafka), Message more than 1000000 byes will be dropped.
 - discard_kafka_delivery_failed - default: false - discard the record where [Kafka::DeliveryFailed](http://www.rubydoc.info/gems/ruby-kafka/Kafka/DeliveryFailed) occurred
-- monitoring_list - default: [] - library to be used to monitor. statsd and datadog are supported
```

Comment:
This parameter doesn't exist for `out_kafka2`

Reply:
You are right, only out_kafka_buffered has it.
Should it be moved to here instead of removing it?:
https://github.com/fluent/fluent-plugin-kafka/blob/009be4e2ad8944f91a35d011c3ca6dfb2a74cebf/README.md?plain=1#L423-L426

Reply:
Sure, I've added it to the `kafka_buffered` section